settings.py

```diff
@@ -13,7 +13,7 @@
 
 DEBUG = True
 TEMPLATE_DEBUG = DEBUG
-DEBUG_PROPAGATE_EXCEPTIONS = False
+DEBUG_PROPAGATE_EXCEPTIONS = True
```

Comment:
I was hitting an error on the log statement before the redirect and there was no stack trace. I assumed this was set to False by accident since it hides stack traces in development.